taxonomies/ingredients.txt

```diff
@@ -26219,7 +26220,7 @@ vegetarian:en:maybe
 # nova:en:hydrolysed-proteins
 <en:protein
 en:hydrolysed proteins, hydrolysed protein
-de:Proteinhydrolysat
+de:Proteinhydrolysat, Würze
```

Comment:
Is Würze not something more generic?

Reply:
Done. Added as new ingredient. Doesn't need to be treated specifically by automatic extraction of substances or products causing allergies or intolerances and food additives, because

> Nach europäischem Lebensmittelrecht kann Würze je nach Art der Herstellung als Aroma eingeordnet werden. In diesen Fällen besteht die Möglichkeit, Würze allgemein als „Aroma“ zu deklarieren. In Deutschland ist dies nicht üblich. In den Zutatenlisten finden sich Verkehrsbezeichnungen wie „Speisewürze“, „Würze“ oder „pflanzliches Eiweißhydrolysat“. Der Hinweis auf die Ausgangssubstanzen Weizen oder Soja erfolgt im Zuge der Allergenkennzeichnung als Hinweis für betroffene Verbrauchergruppen. Der Würze zugesetzte Geschmacksverstärker müssen stets gekennzeichnet werden.

See [Würze (Lebensmittel)](https://de.wikipedia.org/w/index.php?title=W%C3%BCrze_(Lebensmittel)&oldid=190243027). In: Wikipedia, Die freie Enzyklopädie. Bearbeitungsstand: 8. Juli 2019, 17:50 UTC.